src/xinterpreter.cpp

```diff
@@ -117,7 +117,7 @@ void interpreter::execute_request_impl(
     SEXP execution_counter_ = PROTECT(Rf_ScalarInteger(execution_count));
     SEXP silent_ = PROTECT(Rf_ScalarLogical(config.silent));
 
-    SEXP result = r::invoke_hera_fn("execute", code_, execution_counter_, silent_);
+    SEXP result = r::invoke_hera_fn("execute", code_, execution_counter_, silent_, R_GlobalEnv);
```

Comment:
Would it be possible to patch release xeus-r so that it has this. Otherwise, I have to set the argument on the R side and it might not be accepted to CRAN because of that. cc @IsabelParedes @SylvainCorlay

Reply:
I'll try to release with the default in the R code and see if it's accepted.

Reply:
We can release 0.6.1 if needed, no problem there

Reply:
it's not necessary, the package is now on cran without the need for this. 
https://cran.r-project.org/web/packages/hera/index.html